soccer/Robot.hpp

```diff
@@ -177,9 +177,18 @@ class OurRobot : public Robot {
     }
 
     /**
-     * Returns a const reference to the path of the robot.
+     * \brief Returns a const reference to the trajectory of the robot.
      */
-    const Planning::Path& path() { return _context->paths[shell()]; }
+    [[nodiscard]] const Trajectory::Trajectory& trajectory() const {
+        return _context->trajectories[shell()];
+    }
+
+    /**
+     * \brief Returns a mutable reference to the trajectory of the robot.
+     */
+    Trajectory::Trajectory& trajectory_mut() {
+        return _context->trajectories[shell()];
+    }
```

Comment:
Does anything actually use these? AFAIK it was just how motion planning accessed the robot's path, but it should now access the trajectories field in context directly.

Reply:
I kind of wanted to have a convenience function to access `trajectories`

Reply:
It's not actually used anywhere, though, and we're trying to phase out usage of `Robot` by anything on the C++ side in favor of granular access of individual fields in context.